helios-tools/src/main/java/com/spotify/helios/cli/command/ControlCommand.java

```diff
@@ -118,7 +118,7 @@ private boolean run(final Namespace options, final Target target, final PrintStr
                       final BufferedReader stdin)
       throws InterruptedException, IOException {
 
-    final HeliosClient client = Utils.getClient(target, err, username);
+    final HeliosClient client = Utils.getClient(target, err, username, options);
```

Comment:
Just reluctance to add an 8th parameter to that method, especially if it was only passed to the Command to be passed to `Utils.getClient()`.